internal/ast/ast.go

```diff
@@ -232,7 +234,7 @@ func (e *AugAssignExpr) String() string {
 	return e.Left.String() + " " + e.Op.String() + "= " + e.Right.String()
 }
 
-// Increment or decrement expression like x++ or --y.
+// IncrExpr is an increment or decrement expression like x++ or --y.
```

Comment:
I made a lot of judgment calls to avoid stuttering where possible, but sometimes it wasn't.